### ISSUE TITLE
Twitter locking my account occasionally

### ISSUE THREAD

**Issue by sukixiv** [dimdenGD/OldTwitter#706]
**Describe the bug**
After use of OldTwitter for a period of time, Twitter locks my account and asks me to pass an Arkose/Captcha challenge to be able to access Twitter again. I'm (thinking) this is a bug/issue with the extension because I have a separate account I almost exclusively use on my phone and hasn't run into the same locking problem - additionally, there are some reviews left on the Chrome Web Store reporting the same problems.

**Screenshots**
[Attached image here](https://i.imgur.com/kOlPzRK.png), blurred account details for privacy - this is the screen after being locked out and disabling OldTwitter (or accessing through mobile Twitter)

**Browser**
Google Chrome (version 122.0.6261.129)

**Links**
[N/A]

**OldTwitter version**
1.8.2.2

**Comment by typedrat** [comment]
I'm having the same issue.

**Comment by soundplusfury** [comment]
I don't think it's browser related. I had this happen a week or so ago and I use Firefox. (In fact I wonder if Twitter is trying to make some extensions completely unworkable? Sorry to be quite so gloomy.)

**Comment by GJJellyfish** [comment]
I've gotten this problem as well.

**Comment by RFLEpicGamer** [comment]
so, i guess i have to disable this extension for a bit

**Comment by RFLEpicGamer** [comment]
it's been a week, and this problem haven't fixed?

**Comment by SenselessVirus** [comment]
There was an update that said it "hopefully fixed lockouts" recently

It did not in fact fix lockouts.

**Comment by RFLEpicGamer** [comment]
Any updates on this issue?

**Comment by dimdenGD** [comment]
Latest unpublished version of OldTwitter already might have fixed this issue. I'm still waiting for confirmation that it actually works, I myself stopped getting logged out and some people said too but still not completely sure

**Comment by GarnetSunset** [comment]
Still happening to me on the latest update

**Comment by SenselessVirus** [comment]
Do we even know what part of the add-on might be causing the lockouts? Since the issue started I've been running a different twitter add-on(just removes ads and the for you tab, nothing as extensive as this) for comparison and it doesn't cause lockouts.

**Comment by sukixiv** [comment]
Haven't run into the issue since the latest update so we'll see if it sticks. Still using Chrome if that matters.

**Comment by SenselessVirus** [comment]
Just happened to me again. Something that might be tripping it is opening multiple twitter tabs at once, that's what I was doing when it happened.

**Comment by RFLEpicGamer** [comment]
what old twitter extension version are you using rn?

**Comment by SenselessVirus** [comment]
> what old twitter extension version are you using rn?

1.8.3.6 on firefox

**Comment by RFLEpicGamer** [comment]
Does that mean that i have to completely stop using this extension? :(

**Comment by GarnetSunset** [comment]
Turned mine off until its confirmed fixed.

**Comment by infinitephantasm** [comment]
haven't seen any log-out or lock issues since the update (1.8.3.6), at least on firefox

**Comment by RFLEpicGamer** [comment]
So, am i safe to use oldtwitter again?

**Comment by GarnetSunset** [comment]
Still having issues on my end, logouts galore

**Comment by infinitephantasm** [comment]
yeah, unfortunately i spoke too soon, it happened to me just an hour ago -- no arkose verification, though

**Comment by RFLEpicGamer** [comment]
i updated the extension, there are any fixes?

**Comment by RFLEpicGamer** [comment]
am i safe to use this extension again?

**Comment by GarnetSunset** [comment]
Update, still happening.
RFL asking multiple times will not help.

**Comment by RFLEpicGamer** [comment]
Sorry

**Comment by RFLEpicGamer** [comment]
I miss this extension :(

**Comment by dimdenGD** [comment]
then use it. I've been using it fine with no logouts for a month now on chrome

**Comment by RFLEpicGamer** [comment]
Ok, i'll test it now

**Comment by GarnetSunset** [comment]
I got locked out again

**Comment by SenselessVirus** [comment]
Happened to me again as well, at this point this extension is fucked. You're better off using Control Panel for Twitter ( https://github.com/insin/control-panel-for-twitter/ ) instead. It doesn't take it all the way back to 2016 twitter but it has a lot of the same options and doesn't get you locked out of your account constantly.

**Comment by RFLEpicGamer** [comment]
Damn. I guess i can't use this extension anymore

**Comment by megakarlach** [comment]
Same thing happened recently just after installing the extension then disabling it. Right now I passed the captcha and ended up with the "Something went wrong" error afterward.

**Comment by jonbarrow** [comment]
This happened to me once a while ago, but I'm not actually sure it's 100% the extension causing it. I have several alt accounts I use for various separate purposes, and 2 of them I use *exclusively* on mobile. Both of those accounts get these challenges at times, despite having never used them with this extension, and has happened more than once for both (compared to the once when using the extension)

I honestly think part of the blame is on Twitter here, sending out false positives for normal activity. Outside of the one occurance I had a while ago it hasn't happened to me since on any of the other accounts I use

**Comment by RFLEpicGamer** [comment]
> Same thing happened for me recently just after installing the extension (latest version) then disabling it. Right now I passed the captcha and ended up with the "Something went wrong" error afterward.

You disabled it before getting the captcha or after getting the captcha?

**Comment by Sombra-zx** [comment]
the other day's twitter algorithm maked my main account being taken down because this error, this provoked being on a perma ban and doing a full recovery formulary because the evade suspension rule, but was because this bug, so i uninstalled this extension, until they fix this bug, i will not reinstall

**Comment by kokoropaint** [comment]
Pretty sure this extension got my account suspended in its current state (1.8.4 on firefox). Really sucks :/

**Comment by infinitephantasm** [comment]
this extension got me suspended for some reason. i'm leaving it uninstalled until this gets fixed.

**Comment by RFLEpicGamer** [comment]
Thank god that i disabled this extension before it caused that

**Comment by thepwrtank18** [comment]
Still going on. OldTwitter is unsafe. :(

**Comment by typedrat** [comment]
Especially considering my experience in #828.

**Comment by samiscoolest** [comment]
it mainly happens if you search a bunch in a short period of time in my experience

**Comment by megakarlach** [comment]
> > Same thing happened for me recently just after installing the extension (latest version) then disabling it. Right now I passed the captcha and ended up with the "Something went wrong" error afterward.
> 
> You disabled it before getting the captcha or after getting the captcha?

The captcha appears when you trying to unlock your account. On Waterfox browser (fork of Firefox), I got the error, but on Chrome it worked fine and got my account back now.

**Comment by itsTyrion** [comment]
I also had that about once a week and now my account got suspended on a day I did basically nothing, not sure if the suspension is related

**Comment by gotenan** [comment]
Just got locked out for "unusual activity". It wants me to reset my password through some old ass phone number. So I'm likely not getting back in.

**Comment by jamikai** [comment]
this happening again
